src/main/kotlin/racingcar/output/OutputView.kt

```diff
@@ -0,0 +1,22 @@
+package racingcar.output
+
+import racingcar.car.Car
+
+class OutputView {
+    companion object {
+        private val CAR: String = "-"
```

Comment:
```suggestion
        private const val CAR: String = "-"
```
코틀린에서는 상수인 경우에는 const를 붙입니다. 😄